packages/commercetools/composables/src/useCategory/index.ts

```diff
@@ -0,0 +1,27 @@
+import { UseCategory } from '@vue-storefront/interfaces'
+import { ref } from '@vue/composition-api'
+import { getProduct } from '@vue-storefront/commercetools-api'
+
+export default function useCategory (): UseCategory<any, any, any, any, any> {
```

Comment:
why useCategory? shouldn't be useProduct? we want to load and show products not categories yet?